docs/overview/project.rst

```diff
@@ -0,0 +1,128 @@
+
+Visit the :doc:`glossary <../reference-guide/schema-reference>` for more information on the terminologies used in the form.
+
+REST API
+~~~~~~~~
+
+There will be an API to enable direct content upload without using the submission form. A prototype will be available in the near future. If you want to track development you can check it `here <../status.html#capture>`_. A tutorial on using the API will be provided :doc:`here <../tutorials/create-api>` and a reference guide will be found :doc:`here <../reference-guide/api>`.
+
+Connectivity
+~~~~~~~~~~~~
+
+To reduce user effort and ensure accuracy of added information, we are connecting to collaboration databases and systems containing analysis information. This allows us to auto-complete and auto-fill most of an analysis record as soon as it is created, given that the content exists in the databases we link to, as well as to keep the content updated and synchronized.
+
+You can check the status of current connections `here <../status.html#capture>`_. For more details on how autofill works you can go to the tutorial for :doc:`creating an analysis <../tutorials/create>`.
+
+Additionally, there is a possibility for uploading files (e.g. configuration files) as well as providing a URL from which the files are copied and stored automatically.
+
+.. TODO link to file upload tutorial
+
+Versioning
+~~~~~~~~~~
+
+Upon creation of an analysis on CAP it will be assigned a unique identifier. Every time the analysis is edited the new version will be stored as an update to the previous version of the analysis through the identifier system. This will enable references to intermediate analysis steps in the analysis notes, allow keeping track of the analysis and monitoring as well as undoing changes.
+
+Authorisation/Access Control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Authorization on CAP is managed by CERN Single Sign-On, therefore restricting login to CERN members and restricting collaboration access to the collaboration you a member of.
+
+Due to the sensitive nature of research data and analysis information - especially in early stages of an analysis - accessibility of analysis information is subject to permissions set by the creator of an analysis and the researchers involved.
+
+When starting a new analysis, the creator of the analysis draft record will be the only one able to view and edit the record. He can grant read or edit rights to researchers in the analysis team or working group.
+
+As soon as the analysis is submitted, the analysis will be shared with the collaboration meaning its members will acquire rights to view the analysis. Editing rights will remain as they were for the draft.
+A draft version can be submitted any time. There is no drawback in keeping a draft until the analysis is published other than that analysis content is hidden from members of your collaboration.
+
+.. note::
+
+	- only CERN members have access to CAP login
```

Comment:
Well, afaik CERN members will be able to log in (as we use SSO) but won't be able to see or do anything because the information is restricted to collaborations. So I added this for completeness.